sdk/eventhub/event-hubs/samples/expressSample/src/asyncBatchingProducer.ts

```diff
@@ -0,0 +1,198 @@
+import { AbortController, AbortError, AbortSignalLike } from "@azure/abort-controller";
```

Comment:
Looks pretty solid, but we need a sample sample so we know how to use this sample 😁

Reply:
Hi, @bterlson I found your blog https://devblogs.microsoft.com/azure-sdk/how-to-use-abort-signals-to-cancel-operations-in-the-azure-sdk-for-javascript-typescript/. It's quite useful for me to understand abortController. Thanks~ 👍